package.json

```diff
@@ -133,7 +133,7 @@
 		"start-build-web-if-fallback": "node -e \"process.env.DEV_TARGET === 'fallback' && process.exit(1)\" || yarn run start-build-fallback",
 		"start-build-web-if-evergreen": "node -e \"(! process.env.DEV_TARGET || process.env.DEV_TARGET === 'evergreen') && process.exit(1)\" || yarn run start-build-evergreen",
 		"test": "run-s -s test-client test-packages test-server test-build-tools",
-		"test-client": "jest -c=test/client/jest.config.js",
+		"test-client": "TZ=UTC jest -c=test/client/jest.config.js",
```

Comment:
Would it work if we add `process.env.TZ="UTC"` inside `test/client/setup-test-framework.js`?

If it does, I'd say that's a cleaner solution: all test setup is contained inside a single file where we can add comments as needed to explain the reasoning for that setup, allows `yarn jest --config=...` to continue working (I think VSCode Jest extension does it) and will make it easier to refactor in case we want `test-packages` or `test-server` to also use it.

Reply:
> Would it work if we add process.env.TZ="UTC" inside test/client/setup-test-framework.js?

Nope, I first tried that, and it didn't work.

It works if we add it at the top of `jest.config.js`, but to me that's worse than I've suggested here, because it feels as if that env var is oddly bleeding through the config file.